akka-docs-dev/rst/scala/code/docs/http/scaladsl/server/RejectionHandlerExamplesSpec.scala

```diff
@@ -23,13 +23,13 @@ object MyRejectionHandler {
         complete(HttpResponse(BadRequest, entity = "No cookies, no service!!!"))
       }
       .handle { case AuthorizationFailedRejection ⇒
-        complete(Forbidden, "You're out of your depth!")
+        complete((Forbidden, "You're out of your depth!"))
```

Comment:
I don’t know why anyone would want to use auto-tupling, so I cannot tell you why the line previously was as it was ;-) (seriously, though: the associated warning has caught real bugs before and therefore we abide by its imposed restrictions).